common/build/eslint-config-fluid/scripts/cleanup-printed-configs.js

```diff
@@ -8,24 +8,22 @@ const path = require("path");
 const sortJson = require("sort-json");
 
 (async () => {
-    const myArgs = process.argv.slice(2);
-    const sourcePath = myArgs[0] ? myArgs[0] : undefined;
+    const args = process.argv.slice(2);
+    const sourcePath = args[0];
     const files = await fs.promises.readdir(sourcePath);
 
     for (const file of files) {
-        let json;
         let filePath = path.join(sourcePath, file);
-
         let content = fs.readFileSync(filePath);
-        json = JSON.parse(content);
+        let json = JSON.parse(content);
```

Comment:
const?

Reply:
Good point. I thought that since I was deleting a property it needed to be `let` but I see now how wrong that was. References! :)